### DISCUSSION TITLE
How to list installed packages?

### DISCUSSION THREAD

**Discussion by ElSamhaa** [webinstall/webi-installers#868] [General]
How to list installed packages?

**Comment by coolaj86** [comment]
Webi isn't intended to be a full-blown package manager, just an installer, so it doesn't have a specific mechanism to track that.

However, you can search your shell history to see what webi commands you've run:

```sh
grep -r 'webi' ~/.*history* ~/.local/share/
```

Also, Linux has well-defined, standard locations where programs are supposed to install themselves, and Webi follows that whenever possible, so you can see relevant files in these locations:

```sh
ls ~/.local/opt/ ~/.local/bin/ ~/.config/ ~/.local/share/
```

Any installer that follows the Linux / Unix file system standards will install there but, in practice... Webi is one of the few tools that follows the spec correctly, so much of what you see there came from Webi.
